src/sass/_instance_detail_overview.scss

```diff
@@ -0,0 +1,86 @@
+.instance-overview-tab {
+  overflow: auto;
+
+  .p-muted-heading {
+    color: #757575;
+    font-weight: 500;
+  }
+
+  .row {
+    margin-top: $spv--large;
+  }
+
+  .general,
+  .usage,
+  .networks {
+    border-bottom: 1px solid $color-mid-light;
+
+    table {
+      margin-bottom: $spv--large;
+    }
+  }
+
+  .general,
+  .usage {
+    tr {
+      border: none;
+    }
+
+    td,
+    th {
+      padding-bottom: $spv--x-small;
+      padding-left: 0;
+      padding-top: $spv--x-small;
+    }
+
+    th {
+      width: 30%;
+    }
+  }
+
+  .networks,
+  .profiles {
+    th {
+      color: #757575;
+      font-size: 0.85rem;
+      font-weight: 600;
+      letter-spacing: 0.1em;
+    }
+  }
+
+  .networks {
+    th,
+    td {
+      &:nth-child(2) {
+        width: 80px;
+      }
+
+      &:nth-child(3) {
+        width: 94px;
+
+        @include extra-large {
+          width: 105px;
```

Comment:
Better use `rem` than px as a unit.
The type and managed columns are too narrow, when sorting by them the heading gets truncated. A bit more space would be good.

Reply:
We've always been using `px` for column widths - see [these](https://github.com/canonical/lxd-ui/blame/f1dbf916f78f56640231d943527d17de9cf23ff8/src/sass/_instance_list.scss#L107) that you defined for the instance table. For consistency, it doesn't make much sense to stop now all of a sudden. Rather, we could have a separate ticket to convert all column widths from `px` to `rem` in the future, if you like.